common.blocks/control/control.js

```diff
@@ -106,7 +106,9 @@ provide(BEMDOM.decl(this.name, /** @lends control.prototype */{
 }, /** @lends control */{
     live : function() {
         this
-            .liveBindTo('control', 'focusin', this.prototype._onFocus)
+            .liveBindTo('control', 'focusin', function() {
```

Comment:
Why do we have [separate methods in then prototype][1] in all cases expect this one? May be we can add `_onRequestFocus` or `_onFocusin` here just to stay neat?

[1]: https://github.com/bem/bem-components/blob/2e8444a4ad7bf3a859748f377749d9fb65541ae1/common.blocks/select/select.js#L221-L231